configurations/off.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
This would be useful if someone wants to have only small number of rules turned on. Composing a config file with `off.js` and their overrides